docs/project-summary.txt

```diff
@@ -0,0 +1,3 @@
+-- What are we going to do
+-- How are we going to do it (what technologies, what architecture)
```

Comment:
Hey I think this should be different because you suck :c

Reply:
ok, wow, hurtful, but fair